src/Microsoft.DotNet.Build.Tasks/Targets/CodeCoverage.targets

```diff
@@ -6,7 +6,7 @@
   -->
   <PropertyGroup>
     <OpenCoverVersion>4.6.166</OpenCoverVersion>
-    <ReportGeneratorVersion>2.3.0.0</ReportGeneratorVersion>
+    <ReportGeneratorVersion>2.3.1</ReportGeneratorVersion>
     <CoverallsUploaderVersion>1.4</CoverallsUploaderVersion>
```

Comment:
Hmm... I don't think we're doing anything with coveralls anymore.  @mmitche?  Maybe a separate PR should just delete that stuff entirely?